Export Analysis Reports/Sdl.Community.ExportAnalysisReports/Helpers/Help.cs

```diff
@@ -145,13 +169,13 @@ public static bool ReportFileExist(string reportFolderPath)
 			return false;
 		}
 
-		public static void SaveExportPath(string reportOutputPath)
+		public void SaveExportPath(string reportOutputPath)
 		{
 			if (!string.IsNullOrEmpty(reportOutputPath))
 			{
-				if (!Directory.Exists(CommunityFolderPath))
+				if (!Directory.Exists(_communityFolderPath))
```

Comment:
Directory.CreateDirectory(path) creates dir only when it doesn't already exist

Reply:
I've removed the redundant code for all cases where the Directory was created